### ISSUE TITLE
ad-integration-guide.md is imported / generated, but not ignored

### ISSUE THREAD

**Issue by fstanis** [ampproject/amp.dev#3203]
### Current Behavior

`pages/content/amp-dev/documentation/guides-and-tutorials/contribute/vendor-contributions/ad-integration-guide.md` is generated somehow, but not ignored.

### Expected behavior/code

All generated files are gitignored.

### How do we reproduce the issue?

```bash
git checkout future
git reset --hard
git clean -x -d -f
npm i
gulp bootstrap
```

**Comment by matthiasrohmer** [comment]
This would be fixed by #2463, I'll have a look at why this PR is blocked.

**Comment by fstanis** [comment]
Friendly ping, this issue got worse. The following files are now generated or modified by `bootstrap`:

```
pages/content/amp-dev/documentation/guides-and-tutorials/contribute/vendor-contributions/ad-integration-guide.md
pages/content/amp-dev/documentation/guides-and-tutorials/learn/amp-actions-and-events.md
pages/content/amp-dev/documentation/guides-and-tutorials/learn/email-spec/amp-email-components.md
pages/content/amp-dev/documentation/guides-and-tutorials/learn/email-spec/amp-email-format.md
pages/content/amp-dev/documentation/guides-and-tutorials/learn/spec/amphtml.md
pages/shared/data/blog.yaml
pages/shared/data/recent-guides.yaml
pages/content/amp-dev/documentation/guides-and-tutorials/learn/spec/release-schedule.md
```
